src/Icons/doc/index.rst

```diff
@@ -112,9 +112,22 @@ the size of all your icons from a single place.
 Icon Color
 ~~~~~~~~~~
 
-Typically, SVG icons use ``fill="currentColor"`` to inherit the color of the containing element.
-You can set the color in CSS on the container or directly on the SVG element/class.
+By default, SVG icons rendered by UX Icons use ``fill="currentColor"``.
+You can change the icon color in several ways:
 
+You can use the CSS color property to modify your icon through a CSS class.
+
+.. code-block:: twig
+
+    {{ ux_icon('user-profile', {class: 'my_class'}) }}
+
+Or you can just as easily use the color attribute.
+
+.. code-block:: twig
+
+    {{ ux_icon('user-profile', {color: 'red'}) }}
```

Comment:
Setting directly the "color attribute" is [not valid in HTML5](https://html.spec.whatwg.org/multipage/indices.html#attributes-3:attr-link-color) (and was already deprecated in XHTML).

Moreover, current best practices tend to avoid any hard-coded color value, as they do not work good with multiple color modes (dark/light, accessibility, etc.)

So i would suggest we either do not mention this technique directly... or we add some contextual information about it.. wdyt ?